benchmarks/ImageProcessing/BuddyRotate2DBenchmark.cpp

```diff
@@ -0,0 +1,141 @@
+//===- BuddyRotate2DBenchmark.cpp -------------------------------------------===//
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//===----------------------------------------------------------------------===//
+//
+// This file implements the benchmark for Rotate2D operation.
+//
+//===----------------------------------------------------------------------===//
+
+#include <benchmark/benchmark.h>
+#include <buddy/Core/Container.h>
+#include <buddy/DIP/ImageContainer.h>
+#include <buddy/DIP/DIP.h>
+#include <opencv2/opencv.hpp>
+
+using namespace cv; 
+using namespace std;
+
+// Declare input image.
+Mat inputImageBuddyRotate2D;
+
+// Define the angle.
+float BuddyRotate2DAngle;
+
+// Define sizes of input.
+intptr_t sizesInputBuddyRotate2D[2];
+
+// Declare Angle option supported.
+enum AngleOption { ANGLE_DEGREE, ANGLE_RADIAN };
```

Comment:
This is already declared inside `DIP.h`, I don't think you should re-declare it here.

Reply:
The new commit removed the duplicate enum class in both Buddy & OpenCV benchmark design.